README.md

```diff
@@ -339,3 +339,30 @@ The table below describes the current Landing Zone Definitions available in the
     </tr>
   </tbody>
 </table>
+
+## Testing
+
+The Landing Zone Service contains unit and integration tests. These tests are run as part of the CI pipeline during the PR process,
+as well as on merge to `main`.
+
+### Local Testing
+
+```sh
```

Comment:
`service` needs to be replaced with `library` in the `./gradlew` commands and the command to start postgres. Otherwise this all worked for me